packages/test/test-utils/src/testContainerRuntimeFactoryWithDefaultDataStore.ts

```diff
@@ -0,0 +1,47 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ContainerRuntimeFactoryWithDefaultDataStore } from "@fluidframework/aqueduct";
+import { IContainerRuntimeOptions } from "@fluidframework/container-runtime";
+import { IContainerRuntime } from "@fluidframework/container-runtime-definitions";
+import { FluidObject } from "@fluidframework/core-interfaces";
+import { RuntimeRequestHandler } from "@fluidframework/request-handler";
+import {
+	IFluidDataStoreFactory,
+	NamedFluidDataStoreRegistryEntries,
+} from "@fluidframework/runtime-definitions";
+
+export const createContainerRuntimeFactoryWithDefaultDataStore = (
```

Comment:
Is this only needed for back-compat? If so, I would add a comment here saying so and create an issue to remove this later.